modules/contact/include/actions/ContactAction.h

```diff
@@ -113,7 +119,11 @@ class ContactAction : public Action
    */
   void addContactPressureAuxKernel();
   /**
-   * Remove repeated contact pairs in the input file.
+   * Remove repeated contact pairs from boundary pairs.
    */
   void removeRepeatedPairs();
+  /**
+   * Create contact pairs from wrap-around sidesets for axisymmetric bodies.
```

Comment:
More nit-picking about comments -- these don't have to be axisymmetric bodies. Maybe 'periodically repeating bodies'?